### PR TITLE
Feature/bpr 5219 business partner information

### PR DESCRIPTION
**Pull request by jacekWojcikOpusCapita**
Handle the BusinessPartnerInformation route component rendering.
https://opuscapita.atlassian.net/browse/BPR-5219

**Short description**

_We have two endpoints (showing almost identical information):_

SupplierInformation : https://businessnetwork.opuscapita.com/bnp/supplierInformation#
BuyerInformation : https://businessnetwork.opuscapita.com/bnp/buyerInformation#

_We want to have only one endpoint_:

-> BusinessPartnerInformation: https://businessnetwork.opuscapita.com/bnp/businessPartnerInformation#

### CODE REVIEW

**Review by jaconsta — CHANGES_REQUESTED** [review]
Thanks for facilitating the completeness.

File: src/client/containers/Main.react.js [hunk 1]
```diff
@@ -41,42 +39,42 @@ import SubstitutionPeriodAdminEditor from './SubstitutionPeriodAdminEditor';
...
-    {
-        return(
+class Main extends React.Component {
+    render() {
+        return (
             <Containers.ServiceLayout serviceName="bnp">
                 <Route path="/" component={Dispatcher} />
                 <Route path="/buyerDashboard" component={BuyerDashboard} />
-                <Route path="/buyerInformation" component={BuyerInformation} />
+                <Redirect from="/buyerInformation" to="/businessPartnerInformation" />
```
[the remark this answers is not in the document]
Comment (maciej-wakula-opuscapita) [reply src/client/containers/Main.react.js:74]: Hi
I have verified - both /supplierInformation?whatever=value and /buyerInformation?whatever=value are redirecting to /businessPartnerInformation?whatever=value

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
```
[the remark this answers is not in the document]
Comment (maciej-wakula-opuscapita) [reply .eslintrc.json]: I have removed the eslint entirely from this branch

**Comment by maciej-wakula-opuscapita** [comment]
The pull request is failing tests as I have not added businessPartnerInformation (as requested) - although I have it available on my computer so just let me know if I should upload it as well

**Pull request merged**